scipy/stats/tests/test_stats.py

```diff
@@ -2403,15 +2403,23 @@ def test_keepdims(self):
         assert_equal(res.count.ravel(), ref.count.ravel())
         assert res.count.shape == (1, 1)
 
-    def test_gh16952(self):
-        # Check that bug reported in gh-16952 is resolved
+    @pytest.mark.parametrize("nan_policy", ['propagate', 'omit'])
+    def test_gh16955(self, nan_policy):
+        # Check that bug reported in gh-16955 is resolved
```

Comment:
Here and above, the issue reference number is a typo, as discussed in person, but that can be fixed in a follow-up.